packages/hawtio/src/plugins/camel/route-diagram/RouteDiagram.tsx

```diff
@@ -16,6 +16,8 @@ import { CamelNodeData, visualizationService } from '@hawtiosrc/plugins/camel/ro
 import 'reactflow/dist/style.css'
 import './routeDiagram.css'
 import { Page } from '@patternfly/react-core'
+import { routesService } from '@hawtiosrc/plugins/camel/routes-service'
```

Comment:
it's in the same plugin module, so you can use relative path `../routes-service`.